src/components/Card/Video.component.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {
+  CardWrapper,
+  CardHeader,
+  CardHeading,
+  CardBody,
+  CardFieldset,
+  CardDescription,
+} from './Card.style';
+
+export default function CardVideo({ name, description, image }) {
+  return (
+    <CardWrapper>
+      <CardHeader>
+        <img
+          src={image}
+          style={{ margin: 'auto', width: 200, display: 'block' }}
+          alt=""
+        />
```

Comment:
It is better to style this with styled-components to maintain consistency with styles

Reply:
solve it !!